sos/plugins/foreman.py

```diff
@@ -241,8 +241,8 @@ def build_query_cmd(self, query, csv=False):
         return _dbcmd % (self.dbhost, csvformat, quote(query))
 
     def postproc(self):
-        satreg = r"((foreman.*)?(\"::(foreman(.*?)|katello).*)?(::(.*)::.*" \
-              r"(passw|cred|token|secret|key).*(\")?:))(.*)"
+        satreg = r"((foreman.*)?(\"::(foreman(.*?)|katello).*)?((::(.*)::.*" \
+              r"(passw|cred|token|secret|key).*(\")?:)|(storepass )))(.*)"
```

Comment:
Could we not just add `storepass` to `(passw|cred|token|secret|key)` here directly and make the ending chars optional? I.E. `(storepass|passw|cred|token|secret|key)(.*)(\")?(:)?)` or even just `(storepass|passw|cred|token|secret|key)(.*)`?

Reply:
The `storepass` is not prefixed with `::` like the others:

    "::foreman::params::oauth_consumer_secret": pvFHQw7XyVgrZSms9NkhNdRYJ9RrUqL4

While:

    keytool .. -storepass gXuQaTFGj8kEo007CMg3XrZE6tuDGFFB ..

So there is nothing in common with the `storepass` and others (apart of space just before the value to scrub).

Maybe rather have an extra r.e. for the `storepass`?